src/components/club/ProspectusGenerator.tsx

```
@@ -0,0 +1,171 @@
+    clubId: string;
+}
+
+export function ProspectusGenerator({ clubId }: ProspectusGeneratorProps) {
+    const [metrics, setMetrics] = useState<ProspectusMetrics | null>(null);
+    const [isLoading, setIsLoading] = useState(true);
+    const [isGenerating, setIsGenerating] = useState(false);
+    const [pitchText, setPitchText] = useState("");
+    const [selectedTiers, setSelectedTiers] = useState<string[]>([]);
+    const [primaryColor, setPrimaryColor] = useState("#6366f1");
+
+    useEffect(() => {
```

ESLint

[error] Delete `··`

(prettier/prettier)